src/templates/docx/LDCETemplate.ts

```diff
@@ -195,7 +197,13 @@ export function LDCETemplate(passport: Passport): Document {
         fields: [
           {
             name: "Please state under what grounds is the certificate sought",
-            value: get("application.basisOfLawfulness"),
+            value:
+              get("application.basisOfLawfulness") === "Immune"
+                ? `Immune - The property has been in this use since ${
+                    get("proposal.start.date") ||
+                    get("proposal.completion.data")
+                  }`
+                : get("application.basisOfLawfulness"),
```

Comment:
This was the trickiest point of feedback - see thread here: https://opensystemslab.slack.com/archives/C5Q59R3HB/p1698851967728249

I think this is a decent first solution, but if this comes back up then we expect to raise a rather time intensive green content ticket to update `application.basisOfLawfulness` values to better match Secretary of State options set.

Reply:
This makes sense to me!